include/libtorrent/session_handle.hpp

```diff
@@ -43,6 +43,8 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "libtorrent/peer_class_type_filter.hpp"
 #include "libtorrent/session_settings.hpp"
 
+#include "libtorrent/kademlia/dht_storage.hpp"
```

Comment:
I think this header may pull in link-time dependencies on private types. specifically the class dht::item